### PR TITLE
Create 2019-02-11--js-core-dev-team-weekly.md

### PR DESCRIPTION
**Pull request by jacobheun**
- [x] Add recording link

### CODE REVIEW

File: meeting-notes/2019/Q1/2019-02-11--js-core-dev-team-weekly.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+- Done:
+  - Reviewed JS-IPLD API
+  - Trying to make npm-on-ipfs more stable
+  - Tracking down potential memory leak
+  - Merged & released unixfs-exporter https://github.com/ipfs/js-ipfs-unixfs-exporter/pull/16
+  - 
+- Blocked:
+ - N/a
+- Next:
+  - Inaugural Package Managers Weekly Sync
```
Comment (alanshaw) [inline meeting-notes/2019/Q1/2019-02-11--js-core-dev-team-weekly.md]: @achingbrain will this be added to the community calendar for anyone to join or is it private right now?

File: meeting-notes/2019/Q1/2019-02-11--js-core-dev-team-weekly.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+## Notes
+
+### Week Update
+
+@alanshaw
+- Done:
+    - Added `iim ls` command [#4](https://github.com/alanshaw/iim/issues/4#issuecomment-460587508)
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-02-11--js-core-dev-team-weekly.md]: 🌟

File: meeting-notes/2019/Q1/2019-02-11--js-core-dev-team-weekly.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        - <img width="216" alt="screenshot 2019-02-05 at 10 17 37" src="https://user-images.githubusercontent.com/152863/52267509-1ab95280-2931-11e9-94a7-c72c327a53fb.png">
+    - CIDv1 base32 default
+        - MFS [#38](https://github.com/ipfs/js-ipfs-mfs/pull/38)
+        - dag-pb [#115](https://github.com/ipld/js-ipld-dag-pb/pull/115)
+        - unixfs engine [#235](https://github.com/ipfs/js-ipfs-unixfs-engine/pull/235)
+        - unixfs importer [#21](https://github.com/ipfs/js-ipfs-unixfs-importer/pull/21)
+        - js-ipfs [#1868](https://github.com/ipfs/js-ipfs/pull/1868)
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-02-11--js-core-dev-team-weekly.md]: HAAAWT! 🔥

File: meeting-notes/2019/Q1/2019-02-11--js-core-dev-team-weekly.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    - Merged DHT PR 🔥 [#856](https://github.com/ipfs/js-ipfs/pull/856)
+    - Released `js-ipfs 0.35.0-pre.0` [CHANGELOG](https://github.com/ipfs/js-ipfs/blob/master/CHANGELOG.md#0350-pre0-2019-02-11)
+        - WARNING there be 🐉:
+            - "Unable to update lock within the stale threshold"
+            - CPU pegged at 100%
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-02-11--js-core-dev-team-weekly.md]: Why is that? DHT?

Comment (alanshaw) [reply meeting-notes/2019/Q1/2019-02-11--js-core-dev-team-weekly.md]: 🤷‍♂️ Needs investigate

**Pull request merged**